src/components/Widgets/WidgetBase.vue

```diff
@@ -603,6 +155,15 @@ export default {
     hideControls() {
       return this.widget.hideControls;
     },
+    component() {
+      const type = COMPAT[this.widgetType] || this.widget.type;
+      if (!type) {
+        ErrorHandler('Widget type was not found');
+        return null;
+      }
+      // eslint-disable-next-line prefer-template
```

Comment:
This line and string concatenation is to work around eslint/eslint/issues/13394 and babel/babel/issues/10904.